examples/plugin/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use plugin_api::plugin_api as pylib_module;
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+use std::path::Path;
+
+fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
```

Comment:
These bounds shouldn't be necessary
```suggestion
fn main() -> Result<(), Box<dyn std::error::Error>> {
```